### DISCUSSION TITLE
A way to show on phone (browser) an RTSP stream

### DISCUSSION THREAD

**Discussion by Barabba11** [Dash-Industry-Forum/webRTC#21] [Q&A]
Hi! mates! I hope you can help me with this 
I need to show an RTSP stream to the devices (mostly phones) of a local network, I would use browser as the simplest way, no need for http page/frame, I would see video on full screen or maximized windowed.
I have a windows server available on the same LAN, and I would use it, I think the best way is to use WebRTC. Can you please help me with the steps? WHat I have to install on the server? SHould the server has II active? Which code the http page should contain?
Thank you a lot for your kind help!

**Comment by Barabba11** [comment]
Can please somebody help me? Thanks!

**Reply by kenyonja** [reply]
I'd suggest asking about this on Slack in DASH-IF (dashif.slack.com) or video-dev (video-dev.slack.com).

**Reply by Barabba11** [reply]
Thank you for your kind answer, how can I contact them? I've registered in this site slack.com but it's almost impossible to contact them, this site is really not user firendly.
Any other way? Maybe they have development here in github?

My question is so complicated to answer here? Please help me! Thank you!
A very simple way to open on browser a video from rtsp

**Comment by Barabba11** [comment]
I've managed it with go2rtc, a very powerful tool
if somebody is interested can read my small quick guide 
https://github.com/AlexxIT/go2rtc/issues/954#issuecomment-1957995617
